src/server/scripts/Commands/cs_deserter.cpp

```diff
@@ -92,7 +93,7 @@ class deserter_commandscript : public CommandScript
     * .deserter bg add 1h30m
     * @endcode
     */
-    static bool HandleDeserterAdd(ChatHandler* handler, Optional<PlayerIdentifier> player, std::string time, bool isInstance)
+    static bool HandleDeserterAdd(ChatHandler* handler, Optional<PlayerIdentifier> player, Optional<std::string> time, bool isInstance)
```

Comment:
How do you imagine it?

Reply:
What do you mean exactly?
The idea is that it adds the default length of the debuff if no parameter for the time is given.
So it checks if time exists, if it does not, it defaults to 15min for BG and 30min for RDF.
If one is given, it first assumes it to be an Acore TimeString and tries to convert it into seconds. If that fails, it next assumes it to be a pure number and tries to convert it. If that's still 0, false is returned.